.vscode/settings.json

```diff
@@ -16,5 +16,5 @@
     "ylabel",
     "ylim"
   ],
-  "python.formatting.provider": "black"
+  "python.formatting.provider": "autopep8"
```

Comment:
This change I'm not so sure It's a good idea.

Reply:
I think it was an accident from my side, good that you picked it up